mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -733,6 +733,9 @@ msgstr "Are you sure you would like to release {name}?"
 msgid "tuxemon_released"
 msgstr "{name} has been released."
 
+msgid "evolution_confirmation"
```

Comment:
Needs more of a lead in, I think...

"{monster} is trying to evolve. Allow evolution?"

Reply:
Yes, it makes sense.